example/sysinfo/odp_sysinfo.c

```diff
@@ -465,62 +464,42 @@ int main(void)
 	printf("\n");
 	printf("  SHM\n");
 	printf("    max_blocks:           %u\n", shm_capa.max_blocks);
-	printf("    max_size:             %" PRIu64 " MB\n",
-	       shm_capa.max_size / MB);
+	printf("    max_size:             %" PRIu64 " MB\n", shm_capa.max_size / MB);
 	printf("    max_align:            %" PRIu64 " B\n", shm_capa.max_align);
 
 	printf("\n");
 	printf("  POOL\n");
 	printf("    max_pools:            %u\n", pool_capa.max_pools);
 	printf("    buf.max_pools:        %u\n", pool_capa.buf.max_pools);
-	printf("    buf.max_align:        %" PRIu32 " B\n",
-	       pool_capa.buf.max_align);
-	printf("    buf.max_size:         %" PRIu32 " kB\n",
-	       pool_capa.buf.max_size / KB);
-	printf("    buf.max_num:          %" PRIu32 "\n",
-	       pool_capa.buf.max_num);
+	printf("    buf.max_align:        %u B\n", pool_capa.buf.max_align);
```

Comment:
> Shorten printf lines by not using PRIu32 as uint32_t is in practice equivalent to unsigned int.

I think this is quite questionable, especially in an example application. If you keep the change, I would suggest adding a string comparison that PRIu32 is the same as "u" and failing in an error if they are not.

Reply:
Agree that it's not 100% type safe, but such prints have not caused problems in practice. It would be a problem for a system where "unsigned int" is smaller than 32bits. If such systems still exist, may those have other problems as well (to run modern Linux, tools, etc). Also, our CI catches real size errors (e.g. %u with uint64_t).